www/base/src/views/WorkerView/WorkerView.tsx

```diff
@@ -52,22 +54,46 @@ export const WorkerView = observer(() => {
         order: "-buildid",
       }
     }));
+  const workersBuildersQuery = useDataApiQuery(
+    () => workersQuery.getRelated(w => w.getBuilders({query: {order: 'name'}}))
+  );
 
   const [workerForActions, setWorkerForActions] = useState<null|Worker>(null);
 
+  const worker = workersQuery.isResolved() && workersQuery.array.length >= 1 ? workersQuery.array[0] : null;
   const topBarActions: TopbarAction[] = [];
-  if (workersQuery.isResolved() && workersQuery.array.length >= 1) {
+  if (worker !== null) {
     topBarActions.push(
       {
         caption: "Actions...",
         variant: "primary",
         action: () => {
-          setWorkerForActions(workersQuery.array[0]);
+          setWorkerForActions(worker);
         },
       });
   }
   useTopbarActions(topBarActions);
 
+  const renderWorkerBuilders = () => {
+    if (worker === null || !workersBuildersQuery.isResolved()) {
+      return <LoadingSpan />;
+    }
+    const builders = workersBuildersQuery.getParentCollectionOrEmpty(worker.id).array;
+    return (
+      <Table hover striped size="sm">
+        <tbody>
+        {
+          builders.map(builder => (
+            <tr key={builder.name}>
+              <Link to={`/builders/${builder.builderid}`}>{builder.name}</Link>
```

Comment:
Cell contents must be put into <td>

Reply:
Did you see an increase of padding compared to before? The style should be similar to builders table.

Reply:
Yes, 
![image](https://github.com/user-attachments/assets/8f5bc0e8-0e27-4b3f-a167-d3346b13116d)
compared to previously screened [here](https://github.com/buildbot/buildbot/pull/8099#discussion_r1791919269)